schema/mongoose.ts

```diff
@@ -789,6 +789,75 @@ export const Video = new Schema<types.Video>(
   { collection: 'Video', timestamps: { createdAt: 'createdDate', updatedAt: 'updatedDate' } }
 );
 
+export const VideoParticipant = new Schema<types.VideoParticipant>(
+  {
+    name: { type: String, required: true },
+    profileId:  { type: mongoose.Schema.Types.ObjectId, ref: 'Profile', required: true },
+    description: { type: String },
+    status: { type: String, default: 'Active', enum: ['Paused', 'Pending', 'Active', 'Archived'] },
+    data: { type: Object, default: {} },
+    meta: { type: Object, default: {} },
```

Comment:
good, you noticed every schema needs a meta (and many have data). they also all need applicationId:


    applicationId: {
      type: mongoose.Schema.Types.ObjectId,
      ref: 'Application',
      required: true,
    },